README.md

```diff
@@ -37,4 +39,49 @@ For a simplified, but less granular setup, you can use the API Token authenticat
 }
 ```
 
+An optional end_date field is can be added to the config.json 
+This functionality has been added to ease the backfill procedure for a limited time duration for Zendesk
+
+If passed, data would be loaded for date >= start_date and date < end_date
+
+### Sideloading for tickets
+
+Sideloading is a functionality provided by Zendesk to fetch related records in a single request 
+more detail - https://developer.zendesk.com/documentation/ticketing/using-the-zendesk-api/side_loading/
+
+There are two ways in which the records are returned:
+
+1. Within the same object as a column
+2. As a separate object 
+
+Here we have provided support for sideloading only the 1st type i.e. the attributes returned in the same object as additional columns
+
+Sideload supported for tickets:
+1. comment_count
+2. dates
+3. metric_sets
+4. slas
+
+To sideload an object a list can be passed in the metadata for tickets object in the catalog.json file
+
+e.g.
+```json
+"metadata": [
+        {
+          "breadcrumb": [],
+          "metadata": {
+            "table-key-properties": [
+              "id"
+            ],
+            "forced-replication-method": "INCREMENTAL",
+            "valid-replication-keys": [
+              "generated_timestamp"
+            ],
+            sideload-objects: ["comment_count","dates","metric_events","slas"]
+          }
+        }
+```
```

Comment:
I think there's something wrong here? maybe try removing `json` on line 68?

Reply:
fixed it by adding quotes to sideload-objects